src/Directory.Build.targets

```diff
@@ -2,6 +2,29 @@
 <Project>
   <Import Project="Sdk.targets" Sdk="Microsoft.DotNet.Arcade.Sdk" />
   
+  <!--
+    Set InformationalVersion to only include the original version prefix and any prerelease suffix.
+    It will be updated later with commit hash information. Without this setting to the original
+    version prefix, the version would include a patch number that is computed from the build number,
+    which makes it difficult to trace back to the exact release version. This information is also
+    used when invoking the global version option for tools using System.CommandLine.
+  -->
+  <PropertyGroup>
+    <InformationalVersion>$(_OriginalVersionPrefix)</InformationalVersion>
+    <InformationalVersion Condition="'$(VersionSuffix)' != ''">$(InformationalVersion)-$(VersionSuffix)</InformationalVersion>
+  </PropertyGroup>
+
+  <!--
+    Set PackageVersion to only include the original version prefix and any prerelease suffix.
+    Without this setting to the original version prefix, the version would include a patch number
+    that is computed from the build number, which makes it difficult to trace back to the exact
+    release version. Only set this if PackageVersion was not explicitly set.
+  -->
+  <PropertyGroup Condition="'$(PackageVersion)' == ''">
+    <PackageVersion>$(_OriginalVersionPrefix)</PackageVersion>
```

Comment:
How often does the originalversionprefix rev? For example, is there going to be a 5.0.1 when we do a small patch? Or would we expect an update for 6.0 only?

Reply:
`_OriginalVersionPrefix` has the value of `VersionPrefix` in eng/Version.props, so it revs as fast as we want it to. If we do small fixes, it will change because we change `VersionPrefix` (to 5.0.1 in your example). If we do next feature release, it will change because we change `VersionPrefix` (to 5.1.0 or 6.0.0, however we decide to do "the next release").

Reply:
The `_OriginalVersionPrefix` property is Arcade's way of preserving our stated `VersionPrefix` because at this point in the targets file, Arcade has overwritten `VersionPrefix` with whatever version it decided to calculate. If that is confusing, then I can create a new property only known to our repo, which is set from `VersionPrefix` in the eng/Version.props file, and used here.